core/templates/dev/head/domain/sidebar/SidebarStatusServiceSpec.js

```diff
@@ -0,0 +1,52 @@
+describe('SidebarStatusService', function() {
+  var SidebarStatusService;
+
+  beforeEach(module('oppia'));
+  beforeEach(inject(function($injector) {
+    SidebarStatusService = $injector.get('SidebarStatusService');
+  }));
+
+  it('should open the sidebar if it is not open', function() {
+    SidebarStatusService.openSidebar();
+    expect(SidebarStatusService.isSidebarShown()).toBe(true);
+  });
+
+  it('should not open the sidebar if it is open', function() {
+    SidebarStatusService.openSidebar();
+    SidebarStatusService.openSidebar();
+    expect(SidebarStatusService.isSidebarShown()).toBe(true);
+  });
```

Comment:
@DubeySandeep I'm not sure whether including this case will make any difference. What do you think?